### DISCUSSION TITLE
HA 2024.4.3 : bump of pymodbus to v3.6.7 ... a little nervous to try

### DISCUSSION THREAD

**Discussion by tmenguy** [wlcrs/huawei_solar#682] [General]
Hi,

I'm on the [1.4.0 alpha 2] of the great Huawei solar .... hope it won't break it. Anyone has tried? safe to go?

BR
Thomas

**Comment by tmenguy** [comment]
well I tried ... and so far so good

**Reply by Roving-Ronin** [reply]
Two HA's running 1.40a2 and both work fine with this update.
